### PR TITLE
Joe Sandoval masters page

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

- Added Joe Sandoval masters conference page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/masters-conference
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page matches the [copy doc](https://docs.google.com/document/d/1Z63eMCe9o4Onk2ezPFLPDTbI-uJezAOzI-aSpXmZ5lc/edit)
- Find the "Read more about managing a large and diverse infrastructure fleet" link, click it
- See that the page matches the [copy doc](https://docs.google.com/document/d/1Z63eMCe9o4Onk2ezPFLPDTbI-uJezAOzI-aSpXmZ5lc/edit)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2190

### CODE REVIEW

File: templates/masters-conference/index.html [hunk 1]
```diff
@@ -105,8 +105,11 @@ <h2>Scaling and delivering applications with high performance and low friction</
       <p>How do you support a diverse infrastructure that spans six data centres, three continents and the public cloud? Adobe uses open-source technologies, including Ubuntu, Kubernetes and OpenStack, to craft a feature-rich platform that developers can build on to best serve their customers. </p>
 
       <p>
-        <a href="/masters-conference/contact-us" class="p-button--neutral is-inline js-invoke-modal">Get notified about the full keynote presentation</a>
-        <br />&mdash;&nbsp;coming in January
+        <a href="https://www.youtube.com/watch?v=IUgODRm9Soc" class="p-button--neutral"><span class="p-link--external">See the full keynote presentation</span></a>
+      </p>
+
+      <p>
+        <a href="/masters-conference/joe-sandoval">Read more about Extended BPF&nbsp;&rsaquo;</a>
```
Comment (caldav) [inline templates/masters-conference/index.html]: Is the new page still about Extended BPF?

Comment (sowasred2012) [reply templates/masters-conference/index.html]: My bad, fixed

**Comment by sowasred2012** [comment]
@pmahnke this is ready for code & QA review when you have time

**Comment by pmahnke** [comment]
@caldav don't we need to update the homepage takeover as well?

**Pull request merged**